src/lib/MediaQueryContext.tsx

```diff
@@ -11,14 +11,14 @@ const MediaQueryContext = React.createContext({
 
 export interface MediaQueryProviderProps {
   children: React.ReactElement;
-  mediaQueryBreakPoint?: string | boolean;
+  mediaQueryBreakPoint?: string;
```

Comment:
`string | boolean ` is a conscious choice.. If user wants to target a specific condition externally, they can make it true(say they dont want mobile mode on certain tablets, even when size is below certain pixels).. I was thinking its more simple than having 2 separate props

Reply:
How about we call it `responsive` ?

`responsive?: string | boolean;`

Reply:
`breakpoint` is also nice imo

Reply:
Let's call it `breakpoint`

Reply:
@sravan-s while we are here, When I am trying to use `SendBirdApp` or `SendbirdProvider`, Netiher of them seem to respect/accept `mediaQueryBreakPoint` with string or boolean value. I tried passing breakpoint like 400px or true as well. I tried to find example in storybook too, but no luck.

Reply:
Hey @theajr(I assume that you are using uikit using npm), those props  are not open yet.. Or are you using this branch?

Reply:
@sravan-s  I am using v3.4.7, which has `mediaQueryBreakPoint` prop in this version tagged codebase. Do you mean this works only in this branch, not the v3.4.7 yet? Let me know if any help required, will be happy to help.

Reply:
Ah, so yeah, its not on 3.4.7 ~ 

We are doing QA on it ~ will release as 3.5.0 hopefully around end of May; with docs, tutorials etc~
Thanks for offering help 🙇

Reply:
Thank you for clarifying.